core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -4321,12 +4331,12 @@ private Sql checkSubQuery(String sql) {
     RelNode root = builder
         .scan("EMP")
         .filter(
-            builder.call(SqlStdOperatorTable.EQUALS,
-                builder.field("EMPNO"), builder.literal(10)))
```

Comment:
Did you modify the formatting on purpose?

Reply:
Sorry this change was not on purpose

Reply:
done